setup.py

```diff
@@ -23,6 +23,7 @@
         "web3",
         "gql[requests]",
         "requests",
+        "git+https://github.com/BalancerMaxis/bal_tools@v0.0.1",
```

Comment:
i think this needs specification of the package name:
```suggestion
        "bal_tools @ git+https://github.com/BalancerMaxis/bal_tools@v0.0.1",
```

current ci run looks to be failing because of this: https://github.com/BalancerMaxis/bal_addresses/actions/runs/8830597706/job/24244081897?pr=269

Reply:
Hmm.  In the other repos when importing bal_addresses we did so lilke this

`git+https://github.com/BalancerMaxis/bal_addresses@0.9.1`

Do you think this changed because the bal_tools version has a v in front of it?

Reply:
can you give an example of another repo where this type of import occurs?

Reply:
Yeah my theory did not work.  Will force push back to before I tried.

Here is an example:
https://github.com/BalancerMaxis/multisig-ops/blob/main/action-scripts/brownie/requirements-actions.txt

Reply:
`requirements.txt` is not the same as `setup.py`

Reply:
Any other ideas?

Reply:
What's odd is that this run, which imports bal_tools works: https://github.com/BalancerMaxis/bal_addresses/actions/runs/8830601437

Reply:
ci env installs properly now, so indeed the issue was simply my suggestion here

the test workflow still fails but this is due to another issue: https://github.com/BalancerMaxis/bal_addresses/actions/runs/8832127552/job/24248815728